#goal
int8 rotDir
---
#result
bool stopRot
---
#feedback
int8 contRot
